specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -2491,6 +2922,251 @@
                     "$ref": "#/definitions/ExternalSecuritySolutionProperties"
                 }
             ]
+        },
+        "NorthSouthHardeningsPolicySummary": {
+            "description": "North-south hardening policy summary",
+            "properties": {
+                "id": {
+                    "description": "The north-south policy id",
+                    "type": "string"
+                },
+                "active": {

```

Comment:
please make it an enum. Its more descriptive and allows for future addition of states.

Reply:
done